test/controllers/devise_token_auth/confirmations_controller_test.rb

```diff
@@ -61,6 +61,12 @@ def token_and_client_config_from(body)
         test 'User shoud have the Last checkin filled' do
           assert @resource.last_sign_in_at?
         end
+        
+        test 'user already confirmed' do
+          assert @resource.sign_in_count > 0 do
+            assert expiry == (Time.now + Time.now + 1.second).to_i
```

Comment:
The `#assert` method from minitest does not yield the block that is given here!
(See [RDoc for `Minitest::Assertions#assert`](http://www.rubydoc.info/gems/minitest/Minitest/Assertions#assert-instance_method))

To further demonstrate this, even with an exception raised from within the block, the test still passes!
~~~ruby
test 'user already confirmed' do
  assert @resource.sign_in_count > 0 do
    assert expiry == (Time.now + Time.now + 1.second).to_i
    raise "Chunky Bacon!"
  end
end
~~~

Reply:
n/m... I commented here waaay late, and this has been resolved by #1113.